bin/generate-bundle-types

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -ex
+
+REPO=sigstore/cosign
+REF=a8ea54021bb6f850b30ee22bd15e0477d623e00d
+PB_PATH=specs/dotsigstore_bundle/pb
+
+mkdir -p /tmp/defs
+
+curl -L --silent --show-error \
+  -o /tmp/defs/envelope.proto \
+  https://raw.githubusercontent.com/${REPO}/${REF}/${PB_PATH}/envelope.proto
+
+curl -L --silent --show-error \
+  -o /tmp/defs/bundle.proto \
+  https://raw.githubusercontent.com/${REPO}/${REF}/${PB_PATH}/bundle.proto
+
+protoc --plugin=./node_modules/.bin/protoc-gen-ts_proto \
```

Comment:
Does this need to be installed separately? Should we set up a Dockerfile for this?

Reply:
Realised running this in docker is not entirely trivial.. can have a crack at this if helpful as I kind of started poking at this while watching MS training videos 😄  would this be helpful?

Reply:
@feelepxyz Yeah, if you wanna take a crack at this, that would be great. I wasn't sure that if asking the dev to have docker installed was any less burdensome than just installing `protoc` 😆

Reply:
> I wasn't sure that if asking the dev to have docker installed was any less burdensome than just installing `protoc` 

This is a good point, I think docker is probably fairly common though. I noticed that using an older version of protoc generated slightly different output which would not be the case if we pin the version in the docker image.

Will also add some docs/readme on installing docker as a prerequisite for generating the bundle protobufs.